tests/test_data/users/complex_csv_a_b.user

```diff
@@ -0,0 +1 @@
+bhttps://githubb.combabcömplexbabсЮЛОажс$4vz*VçàhxpfCbbmwob

```

Comment:
I did this, because no password did contain any of the usual delimiter or quote characters. But yes, we can remove this now 😆 thanks for pushing one...